app/jobs/sms_sender_number_change_job.rb

```diff
@@ -1,4 +1,6 @@
 class SmsSenderNumberChangeJob < ActiveJob::Base
+  include Rails.application.routes.url_helpers
+  Rails.application.routes.default_url_options[:host] = Figaro.env.mailer_domain_name
   queue_as :sms
```

Comment:
unfortunately, it is. Was getting an error before I added this.

I can try to look into how to avoid this config, though.